src/ui/public/agg_types/buckets/terms.js

```diff
@@ -16,12 +16,16 @@ export default function TermsAggDefinition(Private) {
   const createFilter = Private(AggTypesBucketsCreateFilterTermsProvider);
   const routeBasedNotifier = Private(routeBasedNotifierProvider);
 
-  const aggFilter = ['!top_hits', '!percentiles', '!median', '!std_dev', '!derivative'];
+  const aggFilter = [
+    '!top_hits', '!percentiles', '!median', '!std_dev',
+    '!derivative', '!cumulative_sum', '!moving_avg', '!serial_diff'
```

Comment:
When you merge, I'd add some comment that you are also fixing this somewhat unrelated issue of excluding terms for all parent-aggs.